addons/account/views/account_journal_dashboard_view.xml

```diff
@@ -18,7 +18,6 @@
                         <div t-attf-class="#{!selection_mode ? kanban_color(record.color.raw_value) : ''}">
                             <t t-value="JSON.parse(record.kanban_dashboard.raw_value)" t-set="dashboard"/>
                             <t t-value="record.type.raw_value" t-set="journal_type"/>
-                            <t t-call="AccountDropZoneUploader"/>
```

Comment:
Won't this traceback when the server is restarted without a module update?

Reply:
I don't think it will traceback because the template will still be there, but...the kanbanRecord may end up with 2 dropzones and uploaders.
Just tested... ☝️ is true, but everything works